README.md

```diff
@@ -92,6 +92,7 @@ GPU Top without UI tools has minimal dependencies :
 
 ```
 sudo apt-get install libssl-dev
+sudo -H pip2 install mako
```

Comment:
Sorry that I am not familiar with either meson or CI so I do not know the root cause of the hash issue. I built gputop from a clean git clone.

For the sudo requirement, I wrote it that way because README teaches us to install meson with sudo: `$ sudo pip3 install meson`, and I used the same style. I think we can safely drop it:
```suggestion
pip2 install --user mako
```

Reply:
Installing in the user directory sounds fine.